packages/flutter/lib/src/material/progress_indicator.dart

```diff
@@ -383,6 +383,7 @@ class _LinearProgressIndicatorState extends State<LinearProgressIndicator> with
 class _CircularProgressIndicatorPainter extends CustomPainter {
   _CircularProgressIndicatorPainter({
     this.backgroundColor,
+    double maxArcLength = 3 / 2,
```

Comment:
This could probably be defined as a constant at the top of the file, like `_kMaxArcLength`

Reply:
Done